### PR TITLE
Permits backslash in expressions

### PR DESCRIPTION
**Pull request by lukaszsobek**
Associated Issue: #4551

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* Removed replace of backslashes in expressions.js
* Removed test case referencing the replace in expressions.spec.js

### Screenshots/Videos (OPTIONAL)
screenshot-20171125111327-326x275

### CODE REVIEW

File: src/utils/tests/expressions.spec.js [hunk 1]
```diff
@@ -29,10 +29,6 @@ describe("expressions", () => {
     it("sanitizes 2 quotes", () => {
       expect(sanitizeInput('"3"')).toEqual('"3"');
     });
-
-    it("sanitizes forward slashes", () => {
-      expect(sanitizeInput("foo\\\\")).toEqual("foo\\\\\\\\");
-    });
   });
```
Comment (jasonLaster) [inline src/utils/tests/expressions.spec.js]: lets add the two cases

File: src/utils/tests/expressions.spec.js [hunk 2]
```diff
@@ -29,10 +29,6 @@ describe("expressions", () => {
...
 
   describe("getValue", () => {
```
Comment (jasonLaster) [inline src/utils/tests/expressions.spec.js]: and are there others that we need? perhaps where we want to guard the back slash?

File: src/utils/tests/expressions.spec.js (hunk 1 above)
Comment (codehag) [reply src/utils/tests/expressions.spec.js]: yep, we need to add the two test cases!

  1.  Add N\u{61}N as watch expression
  2.  Add '\u{61}' as watch expression

Expected results:

  1.  NaN
  2.  "a"

Comment (lukaszsobek) [reply src/utils/tests/expressions.spec.js]: Test cases added

File: src/utils/tests/expressions.spec.js (hunk 2 above)
Comment (lukaszsobek) [reply src/utils/tests/expressions.spec.js]: I can't think of any cases where we would need it. If you have any I'm more than happy to add them.

**Pull request merged**